pkg/api/api.go

```diff
@@ -224,6 +224,10 @@ func (hs *HTTPServer) registerRoutes() {
 			orgsRoute.Get("/", Wrap(GetOrgByName))
 		}, reqGrafanaAdmin)
 
+		// Current user role
+		apiRoute.Group("/auth/key", func(keyRoute routing.RouteRegister) {
```

Comment:
I guess it should be `/auth/role`, since this is what we return.

Reply:
Actually we return `ApiKeyDTO`, role is only one field of it. I used the same convention which is used in other API endpoints.